### PR TITLE
Feat/cdh 115/remove unset schema version dev config

### PR DESCRIPTION
**Pull request by nikolatrajkovic994**

### CODE REVIEW

File: src/Command/DeleteAllSchemasCommand.php [hunk 1]
```diff
@@ -29,8 +36,16 @@ public function execute(InputInterface $input, OutputInterface $output): int
     {
         $schemas = $this->schemaRegistryApi->getSubjects();
 
+        $hardDelete = (bool) $input->getOption('hard');
+
         foreach ($schemas as $schemaName) {
             $this->schemaRegistryApi->deleteSubject($schemaName);
+
+            if ($hardDelete) {
+                $this->schemaRegistryApi->deleteSubject(
```
Comment (bajdzun) [inline src/Command/DeleteAllSchemasCommand.php:45]: Do we still need to call `deleteSubject` from above in this case or we can just decide depending on a flag which one should be called?

Comment (nikolatrajkovic994) [reply src/Command/DeleteAllSchemasCommand.php:45]: from: https://docs.confluent.io/platform/current/schema-registry/schema-deletion-guidelines.html
> To accomplish a hard delete of a schema (all versions or a specific version), use this two-step process. You must first soft delete the schema, then hard-delete it.

So we need both calls to actually execute the process of "hard delete of schema".

Comment (bajdzun) [reply src/Command/DeleteAllSchemasCommand.php:45]: Cool, thanks for the info 👍

**Review by bajdzun — APPROVED** [review]

**Comment by nikolatrajkovic994** [comment]
To confirm after this merge there should be new minor version release?
Or at the moment of current v2.2.0, the one should be v2.3.0.

**Pull request merged**